add-on/src/lib/precache.js

```diff
@@ -1,21 +1,21 @@
 'use strict'
 /* eslint-env browser, webextensions */
-// const CID = require('cids')
+// import  CID  from 'cids'
 
-// const Tar = require('it-tar')
-// const pipe = require('it-pipe')
-// const all = require('it-all')
-// const concat = require('it-concat')
+// import  Tar  from 'it-tar'
+// import  pipe  from 'it-pipe'
+// import  all  from 'it-all'
+// import  concat  from 'it-concat'
```

Comment:
do we need these? this got replaced as part of the regex match.

Reply:
Looks like they are all for "downloading CID from public gateway". It's commented out code so I would say it's good to be removed. It's in git history if you ever want to reference it. But @lidel may have more insight

Reply:
Yes, this is legacy code that needs to be replaced  – filled https://github.com/ipfs/ipfs-companion/issues/1102 so it is not blocking necromancy PRs ;)